README.md

```diff
@@ -673,6 +675,14 @@ If you do not want to change the git configuration, you can also use:
 $ git push -u <remote> HEAD
 ```
 
+The following command will push all branches that exist locally and remotely with the same name:
+
```

Comment:
I think you describe the *old* default action for `git push` (which used to be the `current` config of `push.default`): according to https://git-scm.com/docs/git-config#git-config-pushdefault this has changed since Git 2.0 and the default config for `push.default` is `simple`.

Personally I tend to prefer the `upstream` config for `push.default` because its as convenient as `simple`
(both allow one to benefit from the `-u` option of `git push` so that doing
`git checkout -b new-branch; git push -u origin HEAD; git push` will *only* push new-branch to origin/new-branch)
and I sometimes need to push a local branch to a remote branch with a different name, so I do
`git checkout -b fork-master; git push -u fork fork-master:master ; git push`
which is allowed by the `upstream` config for `push.default`.